lib/DirectEditing.js

```diff
@@ -78,6 +80,9 @@ DirectEditing.prototype.close = function() {
   this._active = null;
 
   this.resizable = undefined;
+
+  // restoreFocus API is available from diagram-js@15.0.0
```

Comment:
What about adding an integration test, verifying this continues to work with diagram-js@14?

Reply:
Playbook here: https://github.com/bpmn-io/bpmn-js-properties-panel/blob/main/.github/workflows/CI.yml

(Basically execute the tests twice, once with normal dependency, once against defined diagram-js version).